File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        
        arr1 = [0]*26
        arr2 = [0]*26
        
        for item in s:
            arr1[ord(item)-ord('a')]+=1
        
        for item in t:
            arr2[ord(item)-ord('a')]+=1
        
        return arr1 == arr2

File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) 
    {
        int size = 26;
        int array1[26] = {0};
        int array2[26] = {0};
        int sub = (int)'a';

        for (int i = 0; i < s.size(); i++)
        {
            int ord = (int)s[i];
            array1[ord-sub]+=1;
        }

        for (int i = 0; i < t.size(); i++)
        {
            int ord = (int)t[i];
            array2[ord-sub]+=1;
        }
        
        for (int i = 0; i < size; i++)
            if (array1[i] != array2[i])
                return false;
        return true;
    }
};